features/gebruik_in_lopende_tekst.feature

```diff
@@ -1,189 +1,651 @@
+    | voorvoegsel (02.30)            |                    |
+    | geslachtsnaam (02.40)          | Groen              |
+    | voornamen (02.10)              | Ferdinand Cornelis |
+    | aanduiding naamgebruik (61.10) | V                  |
+    En de persoon heeft een partner met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 999992935 |
+    En de partner heeft de volgende naam gegevens
+    | naam                  | waarde |
+    | voorvoegsel (02.30)   |        |
+    | geslachtsnaam (02.40) | Geel   |
+    En de partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
+    | naam                  | waarde   |
+    | datum aangaan (06.10) | 19580401 |
+    En de persoon heeft een partner met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 999992936 |
+    En de partner heeft de volgende naam gegevens
+    | naam                  | waarde |
+    | voorvoegsel (02.30)   |        |
+    | geslachtsnaam (02.40) | Roodt  |
+    En de partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
+    | naam                  | waarde   |
+    | datum aangaan (06.10) | 19610808 |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 999992934                       |
+    | fields              | naam.gebruikInLopendeTekst      |
+    Dan bevat de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
+    | naam                  | waarde             |
+    | gebruikInLopendeTekst | mevrouw Geel-Groen |
+
+Rule: Bij meerdere huwelijken/partnerschappen die allen ontbonden zijn, worden de naamgegevens van de laatst ontbonden partner gebruikt voor het samenstellen van de gebruikInLopendeTekst
+
+  Scenario: meerdere ontbonden relaties gebruikt de laatst ontbonden relatie
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 999992934 |
+    | geslachtsaanduiding | V         |
+    En de persoon heeft de volgende naam gegevens
+    | naam                           | waarde |
+    | voorvoegsel (02.30)            |        |
+    | geslachtsnaam (02.40)          | Wit    |
+    | voornamen (02.10)              | Jan    |
+    | aanduiding naamgebruik (61.10) | V      |
+    En de persoon heeft een partner met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 999992935 |
+    En de partner heeft de volgende naam gegevens
+    | naam                  | waarde |
+    | voorvoegsel (02.30)   |        |
+    | geslachtsnaam (02.40) | Geel   |
+    En de partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
+    | naam                     | waarde   |
+    | datum aangaan (06.10)    | 19580401 |
+    | datum ontbinding (07.10) | 19601001 |
```

Comment:
datum ontbinding is toch geen gegeven was van aangaanHuwelijkPartnerschap?

Reply:
uiteraard niet. aangepast.

Ik heb ook bij elke partner "ex-" toegevoegd, het is immers geen partner (meer) maar een ex-partner. Voor de automatisering betekent dit dat je daarvoor in partners van /personen kan kijken, maar in /partnerhistorie

```
    En de ex-partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
    | naam                     | waarde   |
    | datum aangaan (06.10)    | 19580401 |
    En de ex-partner heeft de volgende ontbindingHuwelijkPartnerschap gegevens
    | naam                     | waarde   |
    | datum ontbinding (07.10) | 19601001 |
```